tests/commands/got.py

```diff
@@ -0,0 +1,32 @@
+"""
+`got` command test module
+"""
+
+import pytest
+
+from tests.utils import (
+    ARCH,
+    _target,
+    gdb_run_cmd,
+    gdb_start_silent_cmd,
+    GefUnitTestGeneric,
+)
+
+
+@pytest.mark.skipif(ARCH in ("ppc64le",), reason=f"Skipped for {ARCH}")
+class GotCommand(GefUnitTestGeneric):
+    """`got` command test module"""
+
+
+    def test_cmd_got(self):
+        cmd = "got"
+        target = _target("format-string-helper")
```

Comment:
Why are we running this here?

Reply:
this was already there
https://github.com/hugsy/gef/blob/283690ae9bfcecbb3deb80cd275d327c46b276b5/tests/runtests.py#L200